manifest.rb

```diff
@@ -0,0 +1,2 @@
+render "app.yaml.ctmpl"
```

Comment:
so using manifest.rb is a relic of using the configure.rb script, which has to be in the repo for this to work.  I think for indie deploys we may want to go a different route, since configure depends on the configs looking a certain way and having certain environment variables.  instead, we should make all rendering happen within the deploy.sh script and use the script in dsde-toolbox, which allows for any "environment" as long as you pass it in during the docker run.

so to render app.yaml.ctmpl you would put this in `deploy.sh`:
`docker run -v $PWD:/app -e GOOGLE_PROJ=$proj -e SERVICE_VERSION=$version -e INPUT_PATH=/app -e OUT_PATH=/app/output render-templates.sh`
this would render out to a directory called "output" in the root of the repo, and would render all ctmpl files in the directory we mounted as a volume, which is currently the root (`$PWD`).  to clean things up though we would probably want to make a configs directory in the root of the repo itself and then pass that as a volume, rather than the whole repo.

Reply:
https://github.com/broadinstitute/dsde-toolbox/blob/master/render-templates.sh if you want to know how that script works